analyzer/src/main/kotlin/managers/utils/MavenSupport.kt

```diff
@@ -486,7 +486,12 @@ class MavenSupport(workspaceReader: WorkspaceReader) {
             }
         }
 
-        log.warn { "Unable to find '$artifact' in any of ${remoteRepositories.map { it.url }}." }
+        when (artifact.classifier) {
+            "sources" -> Level.INFO
```

Comment:
Just wondering @mnonnenmacher, do we even log this case at all? Or maybe just at `DEBUG`?

Reply:
I find this very useful when reviewing analyzer issues for Maven scans and wouldn't like to have to enable DEBUG to get it.

Reply:
But I maybe thought about the binary artifacts which this change is not about, so yes, for source artifacts DEBUG would probably be fine.

Reply:
done

Reply:
The commit message now needs to be adjusted, too, and the title should be shortened anyway. How about

    MavenSupport: Lower the log level for missing source artifacts

    As the lookup for sources artifacts is just an existence check, do not warn
    if no sources artifact could be found but just log it at debug level.